src/components/Pill.tsx

```diff
@@ -0,0 +1,14 @@
+import { styled } from '@atom-learning/components'
+
+export const Pill = styled('div', {
```

Comment:
Just wondering if this is something that should be in `components`. I'm sure I have seen it somewhere in atom-core

Reply:
Yeah I think so, we have one in our main repo and I've seen them in the marketing site designs. I'll leave this here for now and replace in a future PR.